### ISSUE TITLE
loss function may be not right

### ISSUE THREAD

**Issue by hongguangguo** [suraj-deshmukh/Multi-Label-Image-Classification#4]
https://github.com/suraj-deshmukh/Multi-Label-Image-Classification/blob/master/model.py#L47

your final output is 5 element, It should use categorical_crossentropy

**Comment by suraj-deshmukh** [comment]
this is multi label classification problem so categorical_crossentropy won't work. 
have a look on the links below
https://github.com/fchollet/keras/issues/2166

https://stats.stackexchange.com/questions/207794/what-loss-function-for-multi-class-multi-label-classification-tasks-in-neural-n

**Comment by hongguangguo** [comment]
@suraj-deshmukh  I got it, Thank you !
